src/test/java/com/metamx/metrics/cgroups/SigarPidDiscovererTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017 Metamarkets Group Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.metamx.metrics.cgroups;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SigarPidDiscovererTest
+{
+  @Test
+  public void simpleTest() {
+    Assert.assertNotNull(new SigarPidDiscoverer().getPid());
```

Comment:
Yes, I put this method there as a black hole to make sure the return value is consumed. `new SigarPidDiscoverer().getPid()` alone on a new line also looks confusing.